101-machine-learning-compute-create-computeinstance/azuredeploy.parameters.json

```diff
@@ -8,11 +8,14 @@
     "computeName": {
       "value": "GEN-UNIQUE-8"
     },
-    "adminUserName": {
+    "vmSize": {
+      "value": "Standard_DS3_v2"
+    },
+    "tenantId": {
```

Comment:
remove this and use the defaultValue as indicated above

Reply:
Done